.github/workflows/ci.yml

```
@@ -0,0 +1,33 @@
+        include:
+          - os: ubuntu-24.04
+            target: x86_64-linux-gnu
+          - os: ubuntu-24.04-arm
+            target: aarch64-linux-gnu
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/cache@v4
+        with:
+          path: .zig-cache
+          key: zig-cache-${{ runner.target }}
```

actionlint (1.7.9)

property "target" is not defined in object type {arch: string; debug: string; environment: string; name: string; os: string; temp: string; tool_cache: string}

(expression)